recipes/gerrychain/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "gerrychain" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2d3dc1ff35c6f6c1670918f30e8be4c00faed1178b19501e38bfef09c2f023b5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - geopandas
```

Comment:
`geopandas` is broken now b/c, even though the `conda-forge` stack has `gdal` pinned to `libgdal 2.4.*` a `geopandas` install pulls `libgdal 2.3` from `defaults`. The staged recipes solution could be to use the `strict` channel option (ping @conda-forge/core) or maybe "wait out" the openssl migration and hopefully things will magically work themselves out (until the next migration).

In `geopandas` we can:

- drop the optional dependency on `psycopg2`, which is causing the libgdal downgrade
- over-specify `libgdal 2.4` to force conda to do the right thing.

Ping @jorisvandenbossche.

Reply:
Even with the new `geopandas` that pull less dependencies this still solves for `libgdal 2.3.3` from `defaults` :unamused: 

See https://github.com/conda-forge/osmnx-feedstock/pull/40#issuecomment-470318482 for more info.

I'm not sure if we have any other option besides forcing the `strict` channel option in `conda-forge`.

Reply:
I think that is reasonable to force `strict`, especially as that has been what we've told other people to do.

Reply:
I'll send a PR to conda-smithy and staged-recipes tomorrow to implement that. I "forced" it here on CircleCI just as a test.

Reply:
@maxhully we can force `libgdal 2.4` here to "get this merged" b/c over-specifying the geopandas dependencies will force the right deps. Or we can wait for https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/50. The best solution is of course the latter but if you are in a hurry to get this package in we can do the former.

Reply:
I'm OK with waiting! Let me know if there's anything I can do to help out, and thanks for all your work on this---both my recipe and conda-forge in general!